frontend/src/app/service/type-converter/flowchart-converter.service.ts

```diff
@@ -277,6 +279,49 @@ export class FlowchartConverterService {
     };
   }
 
+  /**
+   * extracts the connector from the given connector id
+   * @param vertexIds given linkSourceIds or linkTargetIds
+   * @param workflowTransformation current workflow
+   * @param isCurrentWorkflowId is true if linkSource operator id = currentWorkflow id
+   */
+  public getConnectorFromOperatorById(
+    vertexIds: VertexIds,
+    workflowTransformation: WorkflowTransformation,
+    isCurrentWorkflowId: boolean
```

Comment:
searchInWorkflowIoInterface
javadoc: if true, search for the connector in the workflow io interface instead of the workflow content operators